common/src/main/java/com/box/l10n/mojito/okapi/filters/HtmlFilter.java

```diff
@@ -77,6 +80,15 @@ public Event next() {
     return next;
   }
 
+  void setEmptyAndNbspAsNotTranslatable(ITextUnit textUnit) {
+    if (markEmptyAndNbspAsNotTranslatable) {
+      String source = textUnit.getSource().toString();
+      if ("".equals(source) || " ".equals(source)) {
```

Comment:
Would `source.trim().isEmpty()` work here rather than requiring the double check?

Reply:
+1 to Mark's comment - as we can also have multiple empty spaces in a string.
Alternatively, if we could observe other whitespace characters and we shouldn't translate strings that only contain whitespaces, StringUtils.isBlank() might be a better fit.

Reply:
it is not showing in Github but one of the test is with a non-breaking space. I'll replace with the escape Unicode character instead. I don't think `isBlank()` accept "nbsp".
 
<img width="574" alt="Screenshot 2023-10-13 at 8 37 25 AM" src="https://github.com/pinterest/mojito/assets/9702182/5e849f51-4b36-44fb-a52c-1a0eb4c66a76">
e